### ISSUE TITLE
Deleting empty text area not possible without adding text into area

### ISSUE THREAD

**Issue by gingerling** [WordPress/gutenberg#10334]
**Describe the bug**
I have mobility issues and keep accidentally clicking the plus icon at the top of a block, which inserts and new empty block above. There is no way to then delete this new block. The only way I have found is to put text (just a random letter) in the block which makes the menu pop up, so I can click the three dots and then remove the block. 

It's a small thing in manyways, however, building one page I did this about 10 times and it was extremely annoying/frustrating. 

**To Reproduce**
1. Create a block, add text to it. 
2. Click on the block then click the plus icon at the centre top of the block - a new, empty block appears. 
3. Try to remove the block.
4. Type a letter, it appears in the block - you can now remove the block using the menu for the block. 

**Expected behavior**
Simple x or similar, to be able to remove an empty block. Should be one click.

**Screenshots**

https://greencloud.gn.apc.org/index.php/s/SDo3PeMeDJ6zwFX

https://greencloud.gn.apc.org/index.php/s/LHCQxHdwCBxbdJ2

https://greencloud.gn.apc.org/index.php/s/LHCQxHdwCBxbdJ2

https://greencloud.gn.apc.org/index.php/s/waws8MkX7fscsXC

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] fedora linux
 - Browser [e.g. chrome, safari] firefox

**Smartphone (please complete the following information):**
 - Version [e.g. 22] 3.9.0

**Comment by kkoppenhaver** [comment]
One way I've been thinking about this would be to put a delete icon in the inserter that shows up when the block has no content.

<img width="996" alt="screen shot 2018-10-13 at 6 01 49 pm" src="https://user-images.githubusercontent.com/2965444/46910879-3deb6c80-cf12-11e8-9c84-3e7f57214a35.png">

<img width="997" alt="screen shot 2018-10-13 at 6 02 57 pm" src="https://user-images.githubusercontent.com/2965444/46910880-42b02080-cf12-11e8-93a0-ff710bbeb0e5.png">

Would be interested in hearing some thoughts on this approach or possible alternatives. Once we decide, I can likely tackle this in the next couple days.

**Comment by gingerling** [comment]
I think that looks great! I'm not sure but I suspect this but issue affects
me more than average so I think the delete option is probably more
preferable than making it harder to add a box by accident.

Could the option be selected by default so hitting enter key (before any
content is added) would delete the box in one click?

Thanks

Anna

**Comment by StaggerLeee** [comment]
I agree there shoud be dashicon trash icon there. Planty of space, even in mobile.

**Comment by aduth** [comment]
The in-between inserter no longer automatically creates a paragraph block. As of #11018, it now shows the default inserter menu. As such, it's my understanding this issue would no longer be relevant? Let me know if it is and it can be reopened.
